impl/src/main/java/org/ehcache/internal/store/tiering/CacheStore.java

```diff
@@ -312,6 +313,12 @@ public void disableStoreEventNotifications() {
     }
   }
 
+  @Override
+  public CacheConfigurationChangeListener getConfigurationListener() {
+//    throw new UnsupportedOperationException("TODO Implement me!");
+    return null;
```

Comment:
I see no general comment on this PR indicating it is a partial fix although this code will end up causing exceptions to happen, notably an NPE when trying to update resources on a tiered cache.